### PR TITLE
#676: Fix typos in coding-conventions asciidoc

### PR DESCRIPTION
**Pull request by leonrohne27**
Fixes: #676

### CODE REVIEW

File: documentation/coding-conventions.adoc [hunk 1]
```diff
@@ -146,10 +146,10 @@ Always use the logger to output messages and never use `System.out` or `System.e
 When catching exceptions always ensure the following:
 
 * Never call `printStackTrace()` method on an exception
-* Either log or wrap and re-throw the entire catched exception.
+* Either log or wrap and re-throw the entire catch exception.
```
[the remark this answers is not in the document]
Comment (jan-vcapgemini) [reply documentation/coding-conventions.adoc:149]:
```suggestion
* Either log or wrap and re-throw the entire catched exception.
```

File: documentation/coding-conventions.adoc [hunk 2]
```diff
@@ -146,10 +146,10 @@ Always use the logger to output messages and never use `System.out` or `System.e
...
 Be aware that the cause(s) of an exception is very valuable information.
 If you loose such information by improper exception-handling you may be unable to properly analyse production problems what can cause severe issues.
-** If you wrap and re-throw an exception ensure that the catched exception is passed as cause to the newly created and thrown exception.
+** If you wrap and re-throw an exception ensure that the catch exception is passed as cause to the newly created and thrown exception.
```
[the remark this answers is not in the document]
Comment (jan-vcapgemini) [reply documentation/coding-conventions.adoc:152]:
```suggestion
** If you wrap and re-throw an exception ensure that the catched exception is passed as cause to the newly created and thrown exception.
```

File: documentation/coding-conventions.adoc [hunk 3]
```diff
@@ -170,7 +170,7 @@ Exception in thread "main" java.lang.IllegalStateException: Something failed
 	at com.devonfw.tools.ide.ExceptionHandling.main(ExceptionHandling.java:14)
 ----
 
-As you can see we have no information and clue what the catched `Exception` was and what really went wrong in `doSomething()`.
+As you can see we have no information and clue what the catch `Exception` was and what really went wrong in `doSomething()`.
```
[the remark this answers is not in the document]
Comment (jan-vcapgemini) [reply documentation/coding-conventions.adoc:173]:
```suggestion
As you can see we have no information and clue what the catched `Exception` was and what really went wrong in `doSomething()`.
```

**Review by hohwille — APPROVED** [review]
@leonrohne27 thanks for your PR. Quite some typos you found and fixed. Nice work👍

**Pull request merged**